sandbox-web/src/main/java/com/exadel/sandbox/controllers/location_controller/CountryController.java

```diff
@@ -25,13 +25,15 @@ public CountryController(CountryService countryService) {
 
```

Comment:
Do you use lombok? @RequiredArgsConstructor allows to avoid manual constructor definition (see above, the beginning of the class)